### PR TITLE
Gpavelar/c01-aws02

### PR DESCRIPTION
**Pull request by gpavelar**
# Description

Exercise: C01-AWS02

Student Name: Gustavo Avelar

> AWS CLI commands for bucket creation, copy files, sync files, create iam users and roles. Acess of s3 bucket from any instance EC2 using ReadOnly permission.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/01class/exercises/c01-aws02/gpavelar/README.MD [hunk 1]
```diff
@@ -0,0 +1,164 @@
+# C01-AWS02
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws02/gpavelar/README.MD:1]: can you rename the file to README.md?

File: classes/01class/exercises/c01-aws02/gpavelar/README.MD [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+# output
+
+ROLE arn:aws:iam::478433196210:role/Test-Role 2020-06-21T13:44:23+00:00 / AROAW6ZGXHCZNG4L6UYON Test-Role
+ASSUMEROLEPOLICYDOCUMENT 2012-10-17
+STATEMENT sts:AssumeRole Allow
+PRINCIPAL ec2.amazonaws.com
+
+## Attach a managed policy to an IAM role
+
+aws iam attach-role-policy --policy-arn arn:aws:iam::aws:policy/ReadOnlyAccess --role-name Test-Role
```
Comment (kikobr82) [inline classes/01class/exercises/c01-aws02/gpavelar/README.MD:81]: can you create your own policy with the least privileges you need to achieve the exercise objective from the instance?

File: classes/01class/exercises/c01-aws02/gpavelar/README.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+      "Sid": "VisualEditor0",
+      "Effect": "Allow",
+      "Action": ["s3:GetObject", "s3:ListBucket"],
+      "Resource": ["arn:aws:s3:::mydevops-bucket", "arn:aws:s3:::mydevops-bucket/*"]
+    }
+  ]
+}
+```
+
+Continuando as alterações e removendo uma policy do IAM ROLE
```
Comment (denstorti) [inline classes/01class/exercises/c01-aws02/gpavelar/README.md:104]: English please :)

Comment (gpavelar) [reply classes/01class/exercises/c01-aws02/gpavelar/README.md:104]: OMG,sorry. I don't know why I wrote in portuguese.

**Pull request merged**